src/cmd_line.rs

```diff
@@ -126,32 +140,33 @@ pub fn handle_command_line_arguments(args: Vec<String>) -> Result<(), String> {
         .chain(cmdline.neovim_args)
         .collect();
 
+    if cmdline._novsync {
+        cmdline.vsync = false;
+    }
+
+    if cmdline._nosrgb {
+        cmdline.srgb = false;
+    }
+
     SETTINGS.set::<CmdLineSettings>(&cmdline);
     Ok(())
 }
 
 #[cfg(test)]
 mod tests {
-    use std::env::set_var;
-    use std::sync::Mutex;
-
-    use lazy_static::lazy_static;
+    use scoped_env::ScopedEnv;
 
     use super::*;
-
-    // Use a mutex to ensure that the settings are initialized and accessed in series
-    lazy_static! {
-        static ref ACCESSING_SETTINGS: Mutex<bool> = Mutex::new(false);
-    }
+    use serial_test::serial;
 
     #[test]
+    #[serial]
```

Comment:
Why do we need this serial_test?

Reply:
For two reasons. 
1. Settings is a global variable. Previously all the tests used a mutex to run serially, but I made it more obvious this way. It would probably be bettet to refactor the tests to use only local variables, but I did not want to make a big change like that here. It took me some time to figure out why my tests were randomly failing, until I figured it out.
2. Some tests modifies the system enviornment variables and the parsing uses that, so again they can’t be run in parallel

Reply:
Yeah, it's better this way.

Would be nice to do something like "just run all tests in serial mode", without requiring `#[serial]` on each test, but i don't see a way to do this (except `--test-threads=1`).

Reply:
https://users.rust-lang.org/t/passing-test-threads-1-to-cargo-test-by-default/87225/13

Looks like there is one option.

But do we have other tests that run in parallel just fine? If so, serial_test for this module seems like good choice. Better than mutex.